lib/siwapp/settings.ex

```diff
@@ -21,25 +21,26 @@ defmodule Siwapp.Settings do
   """
   @spec list_series :: [%Series{}]
   def list_series do
-    Repo.all(Series)
+    Series
+    |> order_by(asc: :id)
```

Comment:
This was because the series are ordered by the last updated one by default. So, when I clicked on the default button of one of them, so it was updated, it automatically changed its position to the top of the table and it was a bit annoying. Now with that order although I change the default series, they don't move.

Reply:
ok then